2do_fe/src/components/TaskDetail/TaskDetail.js

```diff
@@ -38,11 +50,24 @@ export default function TaskDetail({ task }) {
             <Moment format="MMM DD">{task.dueDate}</Moment>
           </div>
         </div>
-        <div className="py-2 justify-self-end pr-14 col-span-2">
-          {task.status === "completed"
-            ? StageButton("complete", "Done")
-            : StageButton("inprogress", "In Progress")}
-        </div>
+        {isShownDelete ? (
+          <div className="py-2 justify-self-end col-span-2 flex">
+            <div className="pr-4">
+              {task.status === "completed"
+                ? StageButton("complete", "Done")
+                : StageButton("inprogress", "In Progress")}
+            </div>
+            <div className="pr-5">
+              <RemoveCircleIcon />
```

Comment:
//Use this
<RemoveCircleIcon 
              onClick={() => {
                dispatch(taskActions.deleteTask(task._id));
                window.location.reload();
              }}/>